app/services/coronavirus/pages/content_builder.rb

```diff
@@ -81,8 +81,6 @@ def sub_sections_data
       page.sub_sections.order(:position).map do |sub_section|
         presenter = Coronavirus::SubSectionJsonPresenter.new(sub_section, page.content_id)
         presenter.output
-      rescue Coronavirus::SubSectionJsonPresenter::MarkdownInvalidError => e
-        raise InvalidContentError, e.message
```

Comment:
Something has gone a bit wrong with the rebase as I think this change is supposed to happen in the "Remove test from content builder" commit.

Reply:
yup, not sure how that happened! Good spot, thanks